evm/chains/mainnet/__init__.py

```diff
@@ -1,8 +1,20 @@
 from eth_utils import decode_hex
 
 from evm import constants
-from evm.chains.chain import Chain
+from evm.vm.forks.homestead.constants import (
+    HOMESTEAD_MAINNET_BLOCK
+)
+from evm.vm.forks.eip150.constants import (
+    EIP150_MAINNET_BLOCK
+)
+from evm.vm.forks.spurious_dragon.constants import (
+    SPURIOUS_DRAGON_MAINNET_BLOCK
+)
+from evm.vm.forks.byzantium.constants import (
+    BYZANTIUM_MAINNET_BLOCK
```

Comment:
I think it might be good to just move these constants to a module here (`evm.chains.mainnet.constants`) since they are actually more specific to the mainnet chain than the forks themselves.

Reply:
Good call. I created one for Ropsten with the Byzantium block, too, since it was in `evm.constants`, but it doesn't seem to be used in the project. Let me know if you want me to exclude it.